src/Features/Core/Portable/Completion/CompletionService.cs

```diff
@@ -1,17 +1,19 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+using System;
 using System.Collections.Generic;
 using System.Threading;
 using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Options;
 using Microsoft.CodeAnalysis.Shared.Extensions;
 
 namespace Microsoft.CodeAnalysis.Completion
 {
     internal static class CompletionService
     {
-        public static IEnumerable<CompletionListProvider> GetDefaultCompletionProviders(Document document)
+        public static IEnumerable<CompletionListProvider> GetDefaultCompletionListProviders(Document document)
         {
-            return document.GetLanguageService<ICompletionService>().GetDefaultCompletionProviders();
+            return document.GetLanguageService<ICompletionService>().GetDefaultCompletionListProviders();
```

Comment:
I hadn't spotted that this would require a TypeScript change. I'll back this particular rename out and save it for a larger change that TypeScript would need to react to.